keyboards/preonic/keymaps/snowskeleton/keymap.c

```diff
@@ -0,0 +1,237 @@
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * | Del  |  F1  |  F2  |  F3  |  F4  |  F5  |  F6  |  -   |  _   |   [  |   ]  |  \   |
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * | Enter|  F7  |  F8  |  F9  |  F10 |  F11 |  F12 |  +   |  =   | Pg Up| Pg Dn|Shift |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |             |      |      |      |      |      |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_LOWER] = LAYOUT_preonic_grid(
+  _______,  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  KC_GRV,   KC_EXLM, KC_AT,   KC_HASH, KC_DLR,  KC_PERC, KC_CIRC, KC_AMPR, KC_ASTR, KC_LPRN, KC_RPRN, KC_DELETE,
+  KC_DEL,   KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_MINS, KC_UNDS, KC_LBRC, KC_RBRC, KC_BSLS,
+  KC_ENTER, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_PLUS, KC_EQL,  KC_LCBR, KC_RCBR, KC_RSFT,
+  _______,  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
+),
+
+
+/* Adjust (Lower + Raise)
+ * ,-----------------------------------------------------------------------------------.
+ * |      |      |      |      |      |      |      |      |      |      |      |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      | Reset| Debug|AudTog|      |      |      |      |INSERT|      |      |  Del |
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * |CAPSLK|      |      |      |AGTogg|      |      |      |      |      |      |      |
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * |      |      |      |DELETE|      |      |      |      |      |      |      |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |             |      |      |      |      |      |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_ADJUST] = LAYOUT_preonic_grid(
+  _______, _______, _______,   _______, _______, _______, _______, _______, _______,   _______, _______, _______,
+  _______, QK_BOOT, DB_TOGG,   AU_TOGG, _______, _______, _______, _______, KC_INSERT, _______, _______, KC_DELETE,
+  KC_CAPS, _______, _______,   _______, _______, AG_TOGG, _______, _______, _______,   _______, _______, _______,
+  _______, _______, KC_DELETE, _______, _______, _______, _______, _______, _______,   _______, _______, _______,
+  _______, _______, _______,   _______, _______, _______, _______, _______, _______,   _______, _______, _______
+)
+
+
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+        case QWERTY_MACOS:
```

Comment:
Just a heads up, if you have the single base layer, you don't really need this macro, at all.

Reply:
good catch! removed in [2d915f4](https://github.com/qmk/qmk_firmware/pull/19357/commits/2d915f433a42455b7c4c8749572cfc0663c3d0d1)